resources/js/Components/PortfolioBreakdown/PortfolioBreakdown.contracts.ts

```diff
@@ -4,17 +4,17 @@ export const GRAPH_STROKE_COLOR_EMPTY = "stroke-theme-secondary-300";
 export const GRAPH_STROKE_COLOR_OTHER = "stroke-theme-secondary-400";
 
 export const GRAPH_STROKE_COLORS = [
-    "stroke-theme-hint-600",
-    "stroke-theme-hint-400",
+    "stroke-theme-primary-600",
+    "stroke-theme-primary-400",
     "stroke-theme-warning-400",
     "stroke-theme-success-400",
     "stroke-theme-danger-400",
     GRAPH_STROKE_COLOR_OTHER,
 ] as const;
 
 export const GRAPH_BACKGROUND_COLORS = [
-    "bg-theme-hint-600",
-    "bg-theme-hint-400",
+    "bg-theme-primary-600",
+    "bg-theme-primary-400",
```

Comment:
maybe we can keep this `hint` as well to avoid colors changing in case of a different primary color 🤔